validators/AgeValidator.php

```diff
@@ -36,6 +37,12 @@ public function init()
      */
     public function validateAttribute($model, $attribute)
     {
+        // Check if the user is a member of the admin group
+        if (Group::getAdminGroup()->isMember($model)) {
```

Comment:
@ArchBlood I think better to use here `$model->user->isSystemAdmin()` instead.
But probably we should use this condition only to skip the disabling, I mean to display the error `You must be at least {age} years old.` even for admins.

Reply:
I see no difference between using either option here as both do the same thing, but you are correct that the display of the error does not happen for members of the admin group.

https://github.com/humhub/humhub/blob/df61147fa637f88a32b1998da589f1aec51b06af/protected/humhub/modules/user/models/User.php#L240-L253

https://github.com/humhub/humhub/blob/df61147fa637f88a32b1998da589f1aec51b06af/protected/humhub/modules/user/models/Group.php#L231-L238

Reply:
I've updated to using `$model->user->isSystemAdmin()` even though both options are possible, but if your option is more preferred then we can go with that one.